broker/lib/fabrik/CfPlatformManager.js

```diff
@@ -114,11 +114,22 @@ class CfPlatformManager extends BasePlatformManager {
       );
   }
 
+  /*
+  {
+    protocol: 'tcp',
+    ips: ['10.11.20.248', '10.11.20.255'],
+    applicationAccessPorts: ['8080']
+  }
+  */
   buildSecurityGroupRules(options) {
+    let portRule = '1024-65535';
+    if (options.applicationAccessPorts && _.size(options.applicationAccessPorts) > 0) {
```

Comment:
Shouldn't the check be `Array.isArray(options.applicationAccessPorts) && _.size(options.applicationAccessPorts) > 0`?

Reply:
Yes. Will update this check. A related query: Should we handle a case where options.applicationAccessPorts is not an array? Ideally this should not happen, but with this check (and even with earlier one) default portRule will be used, if options.applicationAccessPorts is not an array.

Reply:
I will like to not complicate this much. The services must put in an array with one entry in this case. Let's not provide another option of just a single entry for the access ports